packages/rum-core/src/domain/view/trackViews.ts

```diff
@@ -158,6 +169,18 @@ export function trackViews(
       currentView.end({ endClocks: startClocks })
       currentView = startNewView(ViewLoadingType.ROUTE_CHANGE, startClocks, options)
     },
+    setViewContext: (context: Context) => {
+      if (!isExperimentalFeatureEnabled(ExperimentalFeature.VIEW_SPECIFIC_CONTEXT)) {
+        return // TODO: remove this check when the feature is enabled by default
+      }
```

Comment:
**💬 suggestion:** ‏you could remove those experimental feature checks, as the "public api" methods are also behind this experimental feature, we can be sure the it won't be used here.